src/app/legacy/containers/WebVitals/index.jsx

```diff
@@ -44,12 +43,4 @@ const WebVitals = ({ pageType }) => {
   return null;
 };
 
-WebVitals.propTypes = {
-  pageType: string,
-};
-
-WebVitals.defaultProps = {
-  pageType: null,
```

Comment:
should this be applied?

Reply:
Shouldn't be required as it will be `null/undefined` anyway if not supplied.